### DISCUSSION TITLE
Mac build #8109 changes date created, date added, and date modified of Resume and Torrents files

### DISCUSSION THREAD

**Discussion by gradu8ed** [transmission/transmission#3462] [Q&A]
Running MacOS 10.13.6
Been running Transmission 3.0 for years without any issues. Decided to try the new stable build #8109. Paused all open, completed and seeding transfers (approx 1.200) in Transmission 3.00. Downloaded and installed then opened build #6109, as noted there was  a delay in program startup, checked out the preferences and didn't see anything new and quit. Reopened Transmission 3.0 and went to unpause the torrents in process and instead of the 254 files in process there are thousands at which point I quit right away. Now;
library/application support/transmission/resume has 6,367 files and the creation date, date added, and date modified are today, 07/16/2022 between 3:43 PM to 4:13 PM
library/application support/transmission/torrents has 6,392 files and the creation date, date added, and date modified are today, 07/16/2022 between 4:00 PM to 4:08 PM, This file total seems reasonable since it was the results of torrents dating between now and 2015. 
My bad for not backing up the Transmission folder in application support. Any ideas on how to roll back/fix would be welcome.

**Comment by GaryElshaw** [comment]
Really afraid to ask (having once been there), no time machine backup?

**Comment by gradu8ed** [comment]
Nope. Just finishing up a NAS to set time machine up in the next few days. Too little too late I guess.

**Comment by GaryElshaw** [comment]
So sorry, it's a horrible feeling. Not on the scale you've experienced, but I recommend starting with a fresh directory and moving to the nightlies to future proof any moves to 4.0 you might want in the future. 

I've made the switch to using the nightlies and occasionally use 3.0. It seems like that particular bug only effects the first use, and subsequent moves back and forth are unaffected.

**Reply by gradu8ed** [reply]
Hey there, I hope you don't mind but I had a different question that may or may not be related. Like I said in the post I've been using 3.0 since it came out and I had a great system and have downloaded and seeded terabytes of data and lost everything in process and complete and seeding. Time Machine is getting started tonight and fortunately I'm getting some of my in process stuff going back through my browsers history file. The problem I'm running into now is that I've lost all my down and up data connectivity. I've added some new torrents but am getting nothing. In contrast I was reliably getting 1-4 mbps down and 0.5-1.5 mbps up. I've checked and checked and nothing in my settings has changed I've tried it with 3.0 and the nightly build and deleted and rreinstalled both. Any ideas?

Matt

**Reply by GaryElshaw** [reply]
A couple of things to check out: making sure Transmission is in system preferences -> security and privacy -> firewall -> firewall options..

I had all sorts of problems due to my isp and couldn't reliably use the port-forwarding option until i got a static ip and a specifically opened port. If it's possible, in the Transmission network settings, having an open port you know reliably works without being forwarded is the ideal.

**Reply by gradu8ed** [reply]
I have apple firewall turned off  and I use little snitch instead. i'd love to figure out port forwarding but i'm not sure how i'd implement it. Like i said i've got apples firewall turned off, i think i've got little snitch set up as open, then my router, which I also think is open. Finally my vpn which offers an open port for port forwarding but it's generally in the 3100-3300 range which is below transmissions which is between 4800 something to 6400 something and changes. occasionally on no regular schedule. The good news is that things have gotten better, not what it used to be but better.

**Reply by GaryElshaw** [reply]
I'm glad things are a little better. I know different folks here have had issues with different VPN configurations, it would definitely be worth doing a search here and a look around the [Transmission forums](https://forum.transmissionbt.com/) to see if any of their experiences might help optimise it.
